doc/classes/Curve2D.xml

```diff
@@ -102,6 +102,17 @@
 				Cubic interpolation tends to follow the curves better, but linear is faster (and often, precise enough).
 			</description>
 		</method>
+		<method name="sample_baked_with_rotation" qualifiers="const">
+			<return type="Transform2D" />
+			<param index="0" name="offset" type="float" />
+			<param index="1" name="cubic" type="bool" default="false" />
+			<param index="2" name="loop" type="bool" default="true" />
+			<param index="3" name="lookahead" type="float" default="4.0" />
+			<description>
+				Similar with [code]interpolate_baked()[/code]. The the return value is [code]Transform2D[/code], with [code]position[/code] as point position, [code]x[/code] as normal vector, and [code]y[/code] as tangent vector. However when [code]get_backed_length() == 0[/code] the return value is a blank, as there is no reasonable way to caculate the rotation.
+				[code]loop[/code] smooth trangent at the end of the curve, it only works when the begin and end of the curve meet. [code]lookahead[/code] is the distance to a nearby point for caculating tangent vector.
```

Comment:
The description needs some work. Also it would be useful to provide an example.
```suggestion
				Similar to [method sample_baked], but returns [Transform2D] that includes a rotation along the curve. Returns empty transform if length of the curve is [code]0[/code].
				Use [param]loop[/param] to smooth the tangent at the end of the curve. [param]lookahead[/param] defines the distance to a nearby point for calculating the tangent vector.
				[codeblock]
				var transform = curve.sample_baked_with_rotation(offset)
				position = transform.get_origin()
				rotation = transform.get_rotation()
				[/codeblock]
```
(my suggestion does not include "it only works when the begin and end of the curve meet" as I wasn't sure how to word it properly)